### ISSUE TITLE
Accessible uploads

### ISSUE THREAD

**Issue by PVince81** [owncloud/owncloud-design-system#536]

**Comment by marcus-herrmann** [comment]
Goal is the follwing strategy to accomodate improved keyboard and screen reader accessibility:

* After the files have been selected via the "File open" prompt supported by the users operating system, focus should be sent to `<oc-progress>`
* `<oc-progress>` gets `tabindex="-1"` to become programmatically focusable
* `<oc-progress>` should supply information of the progress to screen readers via `aria-valuemin`, `aria-valuemax` and `aria-valuenow`. Since the visual progress bar is labelled in percent values, `aria-valuemin` should be 0, `aria-valuemax` 100 and `aria-valuenow` the current dynamic value. While focus is on the `progress` element, its state will be communicated to supporting screen readers
* After the upload is complete:
   * an [`aria-live` region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) announces that the upload has ended successfully
   * Focus sent to the most resonable place, maybe the `<main>` element
